pkg/machine/keys.go

```diff
@@ -21,7 +29,42 @@ func CreateSSHKeys(writeLocation string) (string, error) {
 	return strings.TrimSuffix(string(b), "\n"), nil
 }
 
+func CreateSSHKeysPrefix(dir string, file string, passThru bool, skipExisting bool, prefix ...string) (string, error) {
+	location := filepath.Join(dir, file)
+
+	_, e := os.Stat(location)
+	if !skipExisting || errors.Is(e, os.ErrNotExist) {
+		if err := generatekeysPrefix(dir, file, passThru, prefix...); err != nil {
+			return "", err
+		}
+	} else {
+		fmt.Println("Keys already exist, reusing")
```

Comment:
nit: We imported logrus already could we use `logrus` here.

Reply:
The machine output, in this case, is part of the UI of an interactive terminal session, and not really logging. We don't want the printed output to be cluttered with timestamps, or for it to be filtered (or redirected other than command line pipping)